src/System.ComponentModel.Primitives/src/System/ComponentModel/DesignOnlyAttribute.cs

```diff
@@ -5,47 +5,42 @@
 namespace System.ComponentModel
 {
     /// <summary>
-    ///    <para>Specifies whether a property can only be set at
-    ///       design time.</para>
+    /// Specifies whether a property can only be set at design time.
     /// </summary>
     [AttributeUsage(AttributeTargets.All)]
     public sealed class DesignOnlyAttribute : Attribute
     {
         /// <summary>
-        ///    <para>
-        ///       Initializes a new instance of the <see cref='System.ComponentModel.DesignOnlyAttribute'/> class.
-        ///    </para>
+        /// Specifies that a property can be set only at design time.
+        /// This <see langword='static '/>field is read-only. 
         /// </summary>
-        public DesignOnlyAttribute(bool isDesignOnly) => IsDesignOnly = isDesignOnly;
+        public static readonly DesignOnlyAttribute Yes = new DesignOnlyAttribute(true);
 
         /// <summary>
-        ///    <para>
-        ///       Gets a value indicating whether a property can be set only at design time.
-        ///    </para>
+        /// Specifies that a property can be set at design time or at run time.
+        /// This <see langword='static '/>field is read-only.
         /// </summary>
-        public bool IsDesignOnly { get; }
+        public static readonly DesignOnlyAttribute No = new DesignOnlyAttribute(false);
 
         /// <summary>
-        ///    <para>
-        ///       Specifies that a property can be set only at design time. This <see langword='static '/>field is read-only. 
-        ///    </para>
+        /// Specifies the default value for the <see cref='System.ComponentModel.DesignOnlyAttribute'/>,
+        /// which is <see cref='System.ComponentModel.DesignOnlyAttribute.No'/>.
+        /// This <see langword='static'/> field is read-only.
         /// </summary>
-        public static readonly DesignOnlyAttribute Yes = new DesignOnlyAttribute(true);
+        public static readonly DesignOnlyAttribute Default = No;
 
         /// <summary>
-        ///    <para>
-        ///       Specifies that a property can be set at design time or at run time. This <see langword='static '/>field is read-only.
-        ///    </para>
+        /// Initializes a new instance of the <see cref='System.ComponentModel.DesignOnlyAttribute'/> class.
         /// </summary>
-        public static readonly DesignOnlyAttribute No = new DesignOnlyAttribute(false);
+        public DesignOnlyAttribute(bool isDesignOnly)
+        {
+            IsDesignOnly = isDesignOnly;
+        }
```

Comment:
This is fine, but I'm curious why you chose to move away from the => syntax?

Reply:
I think its clearer to give assignments their own line, and this explicitly shows that this is a constructor. Also in some of these situations, the line would be too long and be cut off which isn't really the idea with expression bodied members